src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
```

Comment:
- [ ] It would be better if the component `App` is class-based as one of the requirements. 😄 👍

Reply:
Hi @BrianSammit eslint throws an error when using class-based components